src/Microsoft.ML.Data/Evaluators/Metrics/BinaryClassificationMetrics.cs

```diff
@@ -19,7 +19,7 @@ public class BinaryClassificationMetrics
         /// a randomly chosen positive instance higher than a randomly chosen negative one
         /// (assuming 'positive' ranks higher than 'negative').
         /// </remarks>
-        public double Auc { get; }
+        public double AreaUnderTheCurve { get; }
```

Comment:
>AreaUnderTheCurve [](start = 22, length = 17)

Does article necessary?

Reply:
Article is necessary but less visibly pleasing.

---
In reply to: [259525948](https://github.com/dotnet/machinelearning/pull/2700#discussion_r259525948) [](ancestors = 259525948)

Reply:
Actually, if we want to go for `AreaUnderPrecisionRecallCurve`, then we may as well call this `AreaUnderRocCurve`. Please don't spell out RecieverOperatorCharacteristic. Please.

---
In reply to: [259549767](https://github.com/dotnet/machinelearning/pull/2700#discussion_r259549767) [](ancestors = 259549767,259525948)

Reply:
I do not like *the* curve because there are multiple curves we could be talking about in this context. So `AreaUnderRocCurve` as @rogancarr suggests in last communication seems most appropriate. I think this is one area where following .NET guidelines to the letter might be non-beneficial...
 `AreaUnderRecieverOperatorCharacteristicCurve` seems a *little* long, as @rogancarr also suggests. But I don't insist on this.

---
In reply to: [259549930](https://github.com/dotnet/machinelearning/pull/2700#discussion_r259549930) [](ancestors = 259549930,259549767,259525948)